Makefile

```diff
@@ -1,5 +1,5 @@
 PROJ=dex
-ORG_PATH=github.com/liquidlabs-co
+ORG_PATH=github.com/coreos
```

Comment:
I think this file, and others, need this ORG_PATH to liquidlabs-co for the go code to compile locally

Reply:
It's the main reason i moved everything to liquidlabs-co

Reply:
I was able to get it to compile.

```bash
export GOPATH=$HOME/go
go get -u github.com/Masterminds/glide
go get -u github.com/sgotti/glide-vc
make revendor
make docker-image
```